spec/features/user_creates_new_book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User creates a new book' do
+  scenario 'successfully' do
+    visit root_path
+
+    create_book("The Lord of the Rings: The Fellowship of the Ring", "J. R. R. Tolkien" )
```

Comment:
I found the answer here: https://github.com/thoughtbot/handbook/blob/master/developing/hound.md.

I will merge this branch when it is ready and simply move it into the thoughtbot repo.